### PR TITLE
Html css branch

### PR DESCRIPTION
**Pull request by tsheporamantso**
In this project, I have successfully:-

- Setup linters.
- Used Figma template for the background image and other related images
- Positioned and styled images using flexbox
- Set up HTML

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<body>
+    <article>
+        <header>   
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
```
Comment (mckent05) [inline index.html]: - Good job trying to stick to the design template as closely as possible, but I suggest you do not need to include this in your design. They were used in the Figma template to signify a mobile design. Please remove this 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <p class="logo">My Logo</p>
```
Comment (mckent05) [inline index.html]: -You did well using a `header` tag for this part of your page. To make your page more semantically correct, I suggest you wrap your logo and menu-icon in a `nav` tag 🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
+            <p class="logo">My Logo</p>  
+            <h2>I'm Mariane Glad to see you!</h2>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        </header>
+    </article>
```
Comment (mckent05) [inline index.html]: - For smenatic reasons, You do not need to use an `article` tag here, the `header` tag does it 👌. Please fix this 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
+            <p class="logo">My Logo</p>  
+            <h2>I'm Mariane Glad to see you!</h2>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```
Comment (mckent05) [inline index.html]: - To improve your site's HTML structure, I advise that you remove these lines and include them in the section below. The `header` tag practically should contain the site's navigation. You can refer to this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info 👍 👍 
- Also, for semantic reasons, I suggest using an `h1` in place of the `h2` tag, You can refer to this [article](https://www.reliablesoft.net/h1-tag/)
- There are little differences in design between your site and the Figma design, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Figma design:
- Please reference the marked section in the Figma design below
rjrjrj
Your design:
hthth

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @tsheporamantso,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your portfolio looks good 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tsheporamantso) [reply index.html]: Thank you for the feedback, implemented the recommendation as suggested.

File: index.html (hunk 2 above)
Comment (tsheporamantso) [reply index.html]: Thank you for the feedback

File: index.html (hunk 3 above)
Comment (tsheporamantso) [reply index.html]: Done, thank you for the feedback.

File: index.html (hunk 4 above)
Comment (tsheporamantso) [reply index.html]: In applying your final recommendation regarding the border-radius on the left side corner (highlighted in blue), I would have to export the Figma image that includes elements that are not required for the project (Highlighted in yellow). I kindly plea you advise further. 
Screenshot_2

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @tsheporamantso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**